frontend/vite.config.ts

```diff
@@ -112,17 +112,6 @@ export default defineConfig(({ mode }) => {
 		},
 		css: {
 			devSourcemap: true,
-			preprocessorOptions: {
```

Comment:
No longer need this now that both reflame and vite are referring to the pre-built css.